### PR TITLE
[10.0][ADD] Module which allow to track changes on partner bank account

### PR DESCRIPTION
**Pull request by benwillig**
Simply adds a chatter on the partner bank account and track account number, bank and currency changes.

### CODE REVIEW

File: partner_bank_mail_thread/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base',
+        'mail'
+    ],
+    'data': [
+        'views/res_partner_bank.xml',
+    ],
+    'demo': [
```
Comment (rousseldenis) [inline partner_bank_mail_thread/__manifest__.py]: @benwillig Please remove blank entry

**Review by rousseldenis — APPROVED** [review]
Code review. Minor comment

File: partner_bank_mail_thread/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```
Comment (sbidoul) [inline partner_bank_mail_thread/README.rst]: Can you please use the new readme fragments for new modules?

File: partner_bank_mail_thread/models/res_partner_bank.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    acc_number = fields.Char(
+        track_visibility='on_change',
+    )
+    bank_id = fields.Many2one(
+        track_visibility='on_change',
+    )
+    currency_id = fields.Many2one(
+        track_visibility='on_change',
+    )
```
Comment (sbidoul) [inline partner_bank_mail_thread/models/res_partner_bank.py:24]: I suggest tracking partner_id too.

**Comment by sbidoul** [comment]
Also, there is a pep8 error in the manifest.

**Review by novawish — APPROVED** [review]

File: partner_bank_mail_thread/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'name': 'Partner Bank Mail Thread',
+    'summary': """
+        Enable a chatter on the partner bank accounts which will track
+        updates done on a partner bank account""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/partner-contact',
+    'depends': [
+        'base',
```
Comment (NL66278) [inline partner_bank_mail_thread/__manifest__.py:15]: No need for this line. Only need to depend on base if there are no other modules.

File: partner_bank_mail_thread/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_bank_mail_thread/__manifest__.py:3]: Make all your license lines https:

**Review by NL66278 — APPROVED** [review]
:+1:  Minor comments. Nothing blocking. Nice addition.

**Comment by NL66278** [comment]
Going to close and open to see if it solves runbot problem.

**Comment by NL66278** [comment]
runbot is (not for the first time) simply disfunctional. Says build contains warnings but does not keep anly logs, does not show what warnings. We have to assume the warnings are bogus.

**Pull request merged**